src/macho.rs

```diff
@@ -358,6 +537,126 @@ pub struct DyldCacheImageInfo<E: Endian> {
     pub pad: U32<E>,
 }
 
+/// Corresponds to struct dyld_cache_slide_pointer5 from dyld_cache_format.h.
+#[derive(Clone, Copy)]
+pub union DyldCacheSlidePointer5 {
+    pub raw: u64,
+    pub regular: DyldChainedPtrArm64eSharedCacheRebase,
+    pub auth: DyldChainedPtrArm64eSharedCacheAuthRebase,
+}
+
+impl Debug for DyldCacheSlidePointer5 {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        unsafe {
```

Comment:
`unsafe` here is a bit weird; a `Debug` implementation shouldn't ever be something that can fail. If it's infallible that's all right, just weird to see something unsafe around `Debug`.

Reply:
It is apparently always unsafe to access members within a union?

Reply:
ooh, union; yeah. It's effectively a 'transmute' to produce each field when you read it, there's no safety at all guaranteed there.